### DISCUSSION TITLE
Mark down tables

### DISCUSSION THREAD

**Discussion by infotoni** [globaleaks/globaleaks-whistleblowing-software#3705] [Q&A]
Hello everyone, I have the latest version of Globaleaks installed, and in the questionnaires, I have added a checkbox for service acceptance. I have included our privacy policies, data handling, etc. I would need to add a table, but I don't know how to do it. Using the typical Markdown tags doesn't display the table. Is it possible to create it in another way?

Thanks in advance.

**Comment by evilaliv3** [comment]
Hello, which markdown tag are you trying to use?

Here is the syntax that we support (standard markdown): https://github.com/showdownjs/showdown/wiki/Showdown's-Markdown-syntax

**Reply by infotoni** [reply]
I used this one    |------| , I put It on a checkbox input in a questionarie.
